code/code/disc/disc_aegis.h

```diff
@@ -34,16 +34,19 @@ class CDAegis : public CDiscipline
     CSkill skCureParalyze;
     CSkill skSecondWind;
     CSkill skRelive;
+    CSkill skCrusade;
 
     CDAegis()
       : CDiscipline(),
         skSanctuary(), skCureParalyze(),
-        skSecondWind(), skRelive() {
+        skSecondWind(), skRelive(),
+        skCrusade() {
     }
     CDAegis(const CDAegis &a)
       : CDiscipline(a),
         skSanctuary(a.skSanctuary), skCureParalyze(a.skCureParalyze),
-        skSecondWind(a.skSecondWind), skRelive(a.skRelive) {
+        skSecondWind(a.skSecondWind), skRelive(a.skRelive),
+        skCrusade(a.skCrusade) {
```

Comment:
Do these copy constructors / assignment operators do anything useful? Why not just delete the whole bunch of them?

Reply:
This is boilerplate that's in every discipline. I'm going to leave this here for now but I'll remove them all in one seperate commit and make sure nothing breaks.